src/Navigation/Home/index.css

```diff
@@ -0,0 +1,190 @@
+.home {
+  padding: 120px 15px 0px 15px;
+  color: #999999;
+  font-family: sans-serif;
+}
+.top-1 {
+  display: flex;
+  flex-direction: row;
+}
+@media only screen and (max-width: 866px) {
+  .top-1 {
+    flex-direction: column;
+    justify-content: center;
+    padding-left: 30px;
```

Comment:
`home` class has given padding to whole sections, I do not think we need any other sub classes to set left and right padding.